pom.xml

```diff
@@ -683,6 +683,19 @@
                                         </pluginExecution>
                                         <pluginExecution>
                                             <pluginExecutionFilter>
+                                                <groupId>org.commonjava.maven.plugins</groupId>
+                                                <artifactId>directory-maven-plugin</artifactId>
+                                                <versionRange>[0.1,)</versionRange>
```

Comment:
Unless the version range matters, I tend to use `[0,)`, in order to avoid the question: "What's so special about versions after/before 0.1?". Doesn't really matter, though. :+1: